tests/cpp_unit.cpp

```diff
@@ -154,6 +154,12 @@ void test_example_cpp(bool copy_on_write) {
 
     r2.printf();
     printf("\n");
+
+    // test select
```

Comment:
Might as well publish the unit tests for select I was using to diagnose the problem in the 64-bit map